CspSubmissionPlugin.inc.php

```diff
@@ -1341,12 +1341,16 @@ public function submissionfilesuploadform_display($hookName, $args)
 			}elseif($stageId == 4){
 				// Buscar componentes de arquivos específicos para o autor
 				$submissionId = $request->getUserVar('submissionId');
-				$userId = $request->getUserVar('userId');
-				$stageId = $request->getUserVar('stageId');
 				$userStageAssignmentDao = DAORegistry::getDAO('UserStageAssignmentDAO'); /* @var $userStageAssignmentDao UserStageAssignmentDAO */
 				$users = $userStageAssignmentDao->getUsersBySubmissionAndStageId($submissionId, $stageId, 14);
 
-				if($users){
+				while ($user = $users->next()) {
+					if($user->getData('id') == $_SESSION["userId"]){
+						$isAuthor = true;
+					}
+				}
+
+				if($isAuthor){
```

Comment:
A variável é criada com false? Se não for, vale um `isset($isAuthor) && $isAuthor`

Reply:
Ok.